src/components/Approval/Approval.tsx

```diff
@@ -278,12 +299,20 @@ export const Approval = () => {
                   <Text color='gray.500' translation='trade.estimatedGasFee' />
                 </Row.Label>
                 <Row.Value textAlign='right'>
-                  <RawText>
-                    {toFiat(approvalFeeCryptoHuman.times(feeAssetFiatRate ?? 1).toString())}
-                  </RawText>
-                  <RawText color='gray.500'>
-                    {toCrypto(approvalFeeCryptoHuman.toNumber(), sellFeeAsset?.symbol)}
-                  </RawText>
+                  <Skeleton isLoaded={approvalNetworkFeeCryptoHuman !== undefined}>
+                    <RawText>
+                      {approvalNetworkFeeCryptoHuman
+                        ? toFiat(
+                            approvalNetworkFeeCryptoHuman.times(feeAssetFiatRate ?? 1).toString(),
```

Comment:
`feeAssetFiatRate` is a string we get with a `bnOrZero()` call, so the nullish coalescing operator will never evaluate to the right side, and logical or won't work either since we'd never have an empty falsy string here but '0' as a worst case scenario

If the intent is a paranoia check on `feeAssetFiatRate` being 0, this will work:
```suggestion
                            approvalNetworkFeeCryptoHuman.times(bn(feeAssetFiatRate).isZero() ? 1: feeAssetFiatRate).toString(),
```

Reply:
I've added zero rate to the conditional render as zero rate would mean we cant display a valid fee anyway

```
{approvalNetworkFeeCryptoHuman && !bnOrZero(feeAssetFiatRate).isZero()
  ? toFiat(approvalNetworkFeeCryptoHuman.times(feeAssetFiatRate).toString())
  : ''}
```